test/constants.js

```diff
@@ -0,0 +1,6 @@
+exports.USER = {
```

Comment:
but it's not a variable, it is a constant. It's supposed to be in uppercase...

Reply:
yes, but for this case is better to not have a lot of things in upper case, it would make the code more difficult to read.
in the practice we use libraries such as factory-girl with avoid us doing this type of things